### ISSUE TITLE
Instructions

### ISSUE THREAD

**Issue by jhuenges** [jhwoodward/ngapi#1]
Hey, I would love to use your code - but I would need some more instructions on how to use it. Can you elaborate on how to use it?

**Comment by debugpoint136** [comment]
Getting this error  on npm run start -
`
Unhandled rejection Error: Store fields must be an object with field names as keys or a function which returns such an object.
`

**Comment by jhwoodward** [comment]
Hi,
Thanks for your interest in my project
Yes - I will try to elaborate on this when I get some time - hopefully this week / weekend
Out of curiosity - what do you want to use it for ?

**Comment by jhwoodward** [comment]
Hi, not sure why - I'll try to have a look when I get some time this week.

**Comment by jhuenges** [comment]
I would like to use neo4j together with [Apollo](http://www.apollostack.com/) and [Meteor](https://www.meteor.com/) to create some kind of a social network

**Comment by wkolcz** [comment]
I want to use it for a university based matching system to allow others around the school to find each other for coffee or lunch based on interests or job skills. Currently its in Laravel/PHP but I am trying to convert it to React/Redux and would love to have more documentation on how to use this project bridging GraphQL and Neo4J

**Comment by creatzor** [comment]
@jhwoodward any update on this? It seems like this would be really useful! Would you mind explaining the process of how you built 1 fat query from evaluating the graphql request if you're unable to write a guide on how to use this?

**Comment by jthompsonmsm** [comment]
+1 On the documentation. It may be great but we need to know how to use it :)

**Comment by jhwoodward** [comment]
Yes... So sorry - I'm really pushed for time at the moment. Thanks for the feedback. I'll try to address this soon !

**Comment by jhwoodward** [comment]
@albert-the-creator the resolver iterates through the graphql query recursively and builds a cypher query string using the pre-defined schema. You should be able to get an idea of it from looking at the code. When I get some time I'll try to provide some documentation on how to use it, and possibly extend it to optionally infer the schema from an existing database.

**Comment by creatzor** [comment]
@jhwoodward awesome! have you seen this project? https://github.com/neo4j-contrib/neo4j-graphql
